eng/common/internal/Tools.csproj

```diff
@@ -1,5 +1,5 @@
+<!-- Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. -->
```

Comment:
@dotnet/dnceng seems like msbuild is failing do to the changes on this file as the project can't be loaded and restored. Is this a known issue?

Reply:
It is known and should be fixed with dotnet/arcade#7693.

Reply:
it should be solved by this pr: https://github.com/dotnet/arcade/pull/7693
are you still facing this problem?

Reply:
Got it, still hitting it but it is probably just a timing issue, so I'll wait for a new update. THanks for the quick response!